assessment_software/src/app/TextField-Component/TextField.component.html

```diff
@@ -0,0 +1,25 @@
+<!-- {{test}} --> 
+<label for="email-address-icon" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Email</label>
+<div class="relative">
+  <div class="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+    <svg aria-hidden="true" class="w-5 h-5 text-gray-500 dark:text-gray-400" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M2.003 5.884L10 9.882l7.997-3.998A2 2 0 0016 4H4a2 2 0 00-1.997 1.884z"></path><path d="M18 8.118l-8 4-8-4V14a2 2 0 002 2h12a2 2 0 002-2V8.118z"></path></svg>
+  </div>
+  <input
+  [id]="id"
+  [value]="value"
+  [placeholder]="placeholder"
+  [autofocus]="autofocus"
+  [required]="required"
+  [ngClass]="classes"
+  [style.background-color]="variant === 'Filled' ? '#EEE' : 'none'"
+  [style.border]="variant === 'Outlined' ? '1px solid #E0E0E0' : 'none'"
+  [style.borderBottom]="variant === 'Standard' ? 'medium solid #000000' : 'none'"
+  [style.borderRadius]="variant === 'Standard' ? '0px' : 'none'"
+  [style.color]="color"
+  [type]="type"
+  (input)="handleChange($event)"
+  aria-describedby="error ? 'error-desc' : null"
+/>
+</div>
+
+<div *ngIf="error" id="error-desc" style="color: red;">Error</div>
```

Comment:
in this case, if there is error, we should display error message. 

`<div *ngIf="error" id="error-desc" style="color: red;">{{errorMessage}}</div>`

and in the TextField component:

```
export class TextField{
    @Input() id: string = '';
    @Input() value: string = '';
    @Input() variant: 'Standard' | 'Filled' | 'Outlined' = 'Standard';
    @Input() placeholder: string = '';
    @Input() autofocus: boolean = false;
    @Input() classes: string = 'red';
    @Input() color: string = 'black';
    @Input() error: boolean = false;
    @Input() required: boolean = false;
    @Input() type: string = 'email';
    @Input() errorMessage: string = "";

    @Output() onChange = new EventEmitter<string>();
 
}
```

Reply:
@Aben-Bel